### ISSUE TITLE
[Accessibility] If File popup menu is triggered, pressing Escape only once doesn't return focus to score totally

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#13350]
**To Reproduce**

Case 1
1. Select note
2. Press **Alt** to enable File menu in Menu bar 
3. Start to navigate down (with down arrow) through File menu elements
4. Press **Escape** - File button is still under focus rectangle
5.  Press **UP ARROW** several times - the **note selected in step №1 starts to move**. It happens because focus returned to score, but partially it's still on Menu bar as well
6. Now press **right arrow** - **focus returns to Menu bar again** (without pressing Alt)

------------------------

Case 2 (without Escape):
1. Select note
2. Press **Alt** to enable File menu in Menu bar 
3. Press **UP ARROW** several times - the **note selected in step №1 starts to move**

Case 1:

https://user-images.githubusercontent.com/90187801/190151449-2a5680f7-01fc-4486-9c8b-33dc1b033e26.mp4

**Expected behavior**
Here can be two design solutions:
1. If File popup menu is triggered, **pressing Esc returns focus to score totally** (and focus rectangle on File button disappears as well)
2. If File popup menu is triggered, **pressing Esc leaves focus on Menu bar totally**. In this case to return focus to score you need to hit Esc again

**Platform information**
Windows, Linux

**Additional context**
#12075

**Comment by jmwmusic** [comment]
This same behaviour is on all menus not just File Menu.  It does at least announce "Application menu panel" now when you first press Esc which helps alert the user to the fact that they are not out of menus and need to press Esc a second time.  But it would be nice if it closed with the first press of Esc.

**Comment by DmitryArefiev** [comment]
Checked in **4.5.2** (and 4.6 master) on Win10 with Narrator.

I think the behavior is as it should be now, and the issue described in **step №5** no longer occurs.

So now 
1. if you **press Alt** (and don't start navigation with arrows), then press **Esc one time**, it returns to the score
2. if you **press Alt** and **start navigation with arrows** (or press Alt+F), you should press **Esc two times**, to return to the top menu at first and then to the score

https://github.com/user-attachments/assets/bb5e5a13-b5f8-4940-85a1-60e4b8c7d72e

FYI @shoogle

**Comment by DmitryArefiev** [comment]
> But it would be nice if it closed with the first press of Esc.

I think the expected behavior:

1. If you press Alt (and don't start submenu navigation with arrows), then **Esc one time** should return to score 
2. If you already in submenu navigation (File, Edit etc), then you should press **Esc two times** (One for Top main menu and another for score return)

https://github.com/user-attachments/assets/cb27ece6-9438-4654-9aeb-c7b60053081b

**Comment by DmitryArefiev** [comment]
Rechecked in **4.5.2** and **4.6 master** on Win10 with Narrator. The issue **still occurs**

https://github.com/user-attachments/assets/3ce61ce7-e6f0-432b-b423-05371044dadb

The problem can be solved this way: once we press Alt (File becomes selected), the score should not react to any key command (Up arrow or Select All) 

(That's discussable)

**Comment by jmwmusic** [comment]
Case 1 seems to be resolved for me in 4.6 and works exactly as described above with first press of Esc returning to top menu and second press back to score, no use of arrows in any direction after pressing Esc are triggering score movement.
But Case 2 remains unchanged and is still an issue.
